code/components/citizen-server-impl/src/ServerResources.cpp

```diff
@@ -814,7 +814,14 @@ static InitFunction initFunction([]()
 
 			if (resourceName[0] == '[' && resourceName[resourceName.size() - 1] == ']')
 			{
-				for (const auto& resource : g_resourcesByComponent[resourceName])
+				auto category = g_resourcesByComponent.find(resourceName);
+
+				if (category == g_resourcesByComponent.end()) {
```

Comment:
Please, keep the indentation settings (that _should_ be defined in the .clang-format or .editorconfig - why is your editor ignoring these?) in mind. `{` goes on a new line.

Reply:
Apologies, this should be fixed now.